src/base/conversion.rs

```diff
@@ -119,7 +119,7 @@ macro_rules! impl_from_into_asref_1D(
             #[inline]
             fn into(self) -> [N; $SZ] {
                 unsafe {
-                    let mut res: [N; $SZ] = mem::uninitialized();
+                    let mut res: [N; $SZ] = mem::MaybeUninit::uninit().assume_init();
```

Comment:
Per the [`MaybeUninit` docs](https://doc.rust-lang.org/std/mem/union.MaybeUninit.html#initialization-invariant), this updated code does not avoid the undefined behavior that `MaybeUninit` tries to solve.

The correct code would be:
```rust
let mut res: MaybeUninit<[N; $SZ]> = MaybeUninit::uninit();
ptr::copy_nonoverlapping(self.data.ptr(), res.as_mut_ptr() as *mut $N, $SZ);
res.assume_init()
```